### PR TITLE
Created the cart page and styled it according to the figma file. Solves issue #12.

### PR DESCRIPTION
**Pull request by arefalrayes**
Created the cart page and styled it according to the Figma file and also added some assets to general assets. 
relates #12

### CODE REVIEW

File: cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          </section>
+      </main>
+    </header>
+    <footer>
+        <div class="sub-total">
+          </span><h3>SUB TOTAL</h3>
+          <h3 class="pricing" style="font-size:1.5rem;">240$</h3>
+        </div>
+        <p class="note">*shipping charges, taxes and discount codes are calculated at the time of accounting.</p>
+    </footer>
```
Comment (sara219) [inline cart/cart.html]: the subtotal part is not the footer, it's a normal section inside the main tag

File: cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    
+      <button class="buy-btn"><img src="../assets/general-assets/shopping-bag.svg" alt="" > BUY NOW  </button>
+
```
Comment (sara219) [inline cart/cart.html]: this is the footer of this page, wrap it with a footer tag 
```suggestion
<footer>
      <button class="buy-btn"><img src="../assets/general-assets/shopping-bag.svg" alt="" > BUY NOW  </button>
       </footer>
```

Comment (sara219) [reply cart/cart.html]: also, reduce the size of the footer on big screens

Screen Shot 2022-02-08 at 3 49 45 PM

File: cart/cart.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                <p class="counter">1</p>
+                <button class="btn-style">
+                  <img src="../assets/general-assets/Plus.svg" alt="close" />
+                </button>
+              </div>
+              <p class="pricing">$120</p>
+            </div>
+          </section>
+      </main>
+    </header>
```
Comment (sara219) [inline cart/cart.html]: move the closing tag of the header after the nav closing tag in line 28

https://github.com/GSG-CF05/FashionApp-team3/blob/a0617862993b487dae34a3892cf3d0ed785a20c6/cart/cart.html#L26-L28

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by shathakh — APPROVED** [review]

**Review by AlaaEid-1 — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**